contracts/utils/package.json

```diff
@@ -0,0 +1,90 @@
+        "pre_build": "run-s compile generate_contract_wrappers",
+        "test": "yarn run_mocha",
+        "rebuild_and_test": "run-s build test",
+        "test:coverage": "SOLIDITY_COVERAGE=true run-s build run_mocha coverage:report:text coverage:report:lcov",
+        "test:profiler": "SOLIDITY_PROFILER=true run-s build run_mocha profiler:report:html",
+        "test:trace": "SOLIDITY_REVERT_TRACE=true run-s build run_mocha",
+        "run_mocha":
+            "mocha --require source-map-support/register --require make-promises-safe 'lib/test/**/*.js' --timeout 100000 --bail --exit",
+        "compile": "sol-compiler --contracts-dir contracts",
+        "clean": "shx rm -rf lib generated-artifacts generated-wrappers",
+        "generate_contract_wrappers": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output generated-wrappers --backend ethers",
+        "lint": "tslint --format stylish --project . --exclude ./generated-wrappers/**/* --exclude ./generated-artifacts/**/* --exclude **/lib/**/* && yarn lint-contracts",
+        "coverage:report:text": "istanbul report text",
+        "coverage:report:html": "istanbul report html && open coverage/index.html",
+        "profiler:report:html": "istanbul report html && open coverage/index.html",
+        "coverage:report:lcov": "istanbul report lcov",
+        "test:circleci": "yarn test",
+        "lint-contracts": "solhint contracts/**/**/**/**/*.sol"
+    },
+    "config": {
+        "abis": "generated-artifacts/@(IOwnable|Ownable|LibBytes|ReentrancyGuard|SafeMath|TestConstants|TestLibBytes).json"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/0xProject/0x-monorepo.git"
+    },
+    "license": "Apache-2.0",
+    "bugs": {
+        "url": "https://github.com/0xProject/0x-monorepo/issues"
+    },
+    "homepage": "https://github.com/0xProject/0x-monorepo/contracts/utils/README.md",
+    "devDependencies": {
+        "@0x/contracts-test-utils": "^1.0.0",
+        "@0x/abi-gen": "^1.0.17",
+        "@0x/dev-utils": "^1.0.19",
+        "@0x/sol-compiler": "^1.1.14",
+        "@0x/sol-cov": "^2.1.14",
+        "@0x/subproviders": "^2.1.6",
+        "@0x/tslint-config": "^1.0.10",
+        "@types/bn.js": "^4.11.0",
+        "@types/lodash": "4.14.104",
+        "@types/node": "*",
+        "@types/yargs": "^10.0.0",
+        "bn.js": "^4.11.8",
+        "chai": "^4.0.1",
+        "chai-as-promised": "^7.1.0",
+        "chai-bignumber": "^2.0.1",
+        "dirty-chai": "^2.0.1",
+        "make-promises-safe": "^1.1.0",
+        "ethereumjs-abi": "0.6.5",
+        "mocha": "^4.1.0",
+        "npm-run-all": "^4.1.2",
+        "shx": "^0.2.2",
+        "solc": "^0.4.24",
+        "solhint": "^1.2.1",
+        "tslint": "5.11.0",
+        "typescript": "3.0.1",
+        "yargs": "^10.0.3"
```

Comment:
Are all of these in use?

Reply:
Not all of them, but I'll use the script in monorepo-scripts to clean it up after I'm finished with this group of PRs